.github/workflows/pydpf-post.yml

```diff
@@ -0,0 +1,78 @@
+name: PyDPF-Post testing
```

Comment:
@PProfizi could we not use the action already written in the post? It looks like the same thing.
Using something like:

      - name: "Test"
        uses: pyansys/pydpf-post/test-post@v1.0
        with:
          python-version: ${{ matrix.python-version }}
          ANSYS_VERSION: ${{env.ANSYS_VERSION}}
          PACKAGE_NAME: ${{env.PACKAGE_NAME}}
          PyDPF-Post_branch: 'master'

Reply:
@cbellot000 I wanted to minimize the use of external actions first for easier debugging, but yes a composite action could definitely be used, although there is none in the Post repo right now.